.eslintrc

```diff
@@ -1,12 +1,24 @@
 {
-  "extends": "airbnb-base/legacy",
```

Comment:
Yeah just use `eslint-config-airbnb`, the standard one.

Yeah the consistent quoted property names sounds like a good idea, and I don't like spaces before function parentheses so whichever way you want to go there is fine with me!